src/NetworkingServer/NeoServer.Networking.Handlers/LogIn/PlayerLogInHandler.cs

```diff
@@ -86,6 +93,17 @@ private async Task Connect(IConnection connection, PlayerLogInPacket packet)
             Disconnect(connection, "Your account is banned.");
             return;
         }
+        
+        if (!_waitingQueueManager.CanLogin(playerRecord, out uint currentSlot))
+        {
+            var retryTime = _waitingQueueManager.GetTime(currentSlot);
+            var message = $"Muitos jogadores online.\nVocê está no lugar {currentSlot} na lista de espera";
```

Comment:
translate to english please base on this
![image](https://github.com/user-attachments/assets/bb55f9f0-487a-4a8e-895d-bf7d7780567f)